<launch>
	<arg name="veh"/>
	<arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman"/>
	<arg name="pkg_name" default="lane_control_adjuster" doc="name of the package"/>
	<arg name="node_name" default="lane_control_adjuster_node" doc="name of the node"/>

        <group ns="$(arg veh)">
	    <node  pkg="$(arg pkg_name)" type="$(arg node_name).py" name="$(arg node_name)" output="screen">
	        <rosparam command="load" file="$(find lane_control_adjuster)/config/$(arg node_name)/$(arg param_file_name).yaml"/>
            <remap from="lane_control_adjuster_node/mode" to="fsm_node/mode"/>
            <remap from="lane_control_adjuster_node/set_state" to="fsm_node/set_state"/>
            <remap from="lane_control_adjuster_node/switcher" to="apriltag_detector_node/switcher"/>


	    </node>
	</group>
</launch>
